metrics.go

```diff
@@ -43,6 +44,10 @@ func initMetrics() {
 		Name: "cachenator_http_requests_total",
 		Help: "Total number of HTTP requests",
 	}, []string{"method", "path", "status"})
+	jwtRequestsMetric = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Maybe worth adding a metric for failed auth attempts too? As a count so you can alert on that.

Reply:
This already has a label for success (true/false). If false, we can also check the type of error in the other label (only 6 types of errors)